test/unit/tech/flash.test.js

```diff
@@ -177,3 +186,28 @@ test('canHandleSource should be able to work with src objects without a type', f
   equal('maybe', canHandleSource({src: 'test.video.flv'}), 'should guess that it is a flash video');
   equal('', canHandleSource({src: 'test.video.wgg'}), 'should return empty string if it can not play the video');
 });
+
+test('seekable should be for the length of the loaded video', function() {
+  let duration = 23;
+
+  // mock out duration
+  tech.el().vjs_getProperty = function(name) {
```

Comment:
@dmlap Instantiating the flash tech in tests I think is what's leading to some of these vjs_getProperty jigglers. Probably because flash is either not supported or never displayed. Would it be good enough to just test the prototype function directly?

```
var mockFlash = {
  duration: function() {
    return this.duration_;
  }
};

mockFlash.duration_ = 23;
let result = Flash.prototype.seekable.call(mockFlash);
// etc.
```

Would that cover everything we need it to?